proposals/locale.md

```diff
@@ -0,0 +1,62 @@
+# Locale
+
+## Problem
+Breaking our monolithic application into several different applications has many benefits. It also introduces many levels of complexity.
+The currently selected locale should be managed in an application context however it can be changed in controls that live in a library context.
+
+- We need to know what the currently selected locale is.
+- We need to know what to do once the locale is changed
+
+
+## Solution:
+### Determining the current locale
+At festicket we rely on the url to determine the currently selected locale.
+Whilst this is not perfect this **is** a business requirement.
+On an application level we will need a server-side middleware that can query the festicket API
```

Comment:
Will this need to be a standalone API endpoint or can we merge it with the 'current user details' request?

Reply:
I don't actually know. Depends whats faster, ideally we want to defer loading the user until we are in the client context. I suppose we could add it to the user and then just rewrite the url on the client once we get the user object back and there is a mismatch. The API will provide translated content either way.

 /cc @mjjared

Reply:
Ah yes